projects/onto-search/src/lib/onto-search-facet/range-facet/range-histogram/range-histogram.component.ts

```diff
@@ -34,6 +42,13 @@ export class RangeHistogramComponent implements AfterViewInit {
     observer.observe(this.canvas.nativeElement);
   }
 
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.data?.currentValue && this.canvas) {
```

Comment:
I don't like this guard condition. You check something which has no obvious relation with the code inside the block. It looks hacky to me.

Reply:
I removed the canvas check. The canvas must be initialized before init() method is called.
Added null check in init() method.